ci/build-msys2.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh -e
+
+if [ "$1" = "meson" ]; then
+    meson setup build_meson      \
+      -D cdda=enabled            \
+      -D d3d-hwaccel=enabled     \
+      -D d3d11=enabled           \
+      -D dvdnav=enabled          \
+      -D egl-angle-win32=enabled \
+      -D jpeg=enabled            \
+      -D lcms2=enabled           \
+      -D libarchive=enabled      \
+      -D libbluray=enabled       \
+      -D libmpv=true             \
+      -D libplacebo=enabled      \
+      -D lua=luajit              \
+      -D pdf-build=enabled       \
+      -D rubberband=enabled      \
+      -D shaderc=enabled         \
+      -D spirv-cross=enabled     \
+      -D tests=true              \
+      -D uchardet=enabled        \
+      -D vapoursynth=enabled     \
+      -D vulkan=enabled
+    meson compile -C build_meson --verbose
+    cp ./build_meson/generated/mpv.com ./build_meson
```

Comment:
Sorry, more nagging. Can you add the `meson test -C build` after this since I just merged it?

Reply:
done